src/module_library/thermal_time_and_frost_senescence.h

```diff
@@ -283,7 +283,8 @@ void thermal_time_and_frost_senescence::do_operation() const
     // linearly with temperature.
     bool const sene_started = TTc >= seneLeaf;
     bool const in_north = lat >= 0.0;
-    bool const late_year = time >= 180.0;
+    int const doy = 1 + std::floor(time / 24);
```

Comment:
Since this module now uses `std::floor`, which is part of the `cmath` header library, this file should have `#include <cmath> // for std::floor` up at the top, just before the includes for `module.h` and `state_map.h`.